src/BSON/PackedArray.c

```diff
@@ -401,6 +421,48 @@ static HashTable* php_phongo_packedarray_get_properties(phongo_compat_object_han
 	return php_phongo_packedarray_get_properties_hash(object, false, 1);
 }
 
+zval* php_phongo_packedarray_read_dimension(phongo_compat_object_handler_type* object, zval* offset, int type, zval* rv)
+{
+	php_phongo_packedarray_t* intern;
+
+	intern = Z_OBJ_PACKEDARRAY(PHONGO_COMPAT_GET_OBJ(object));
+
+	if (Z_TYPE_P(offset) != IS_LONG) {
```

Comment:
Do we rely on PHP to convert numeric strings to integers?

I noticed the one error test only uses `'foo'`, so it might be helpful to add some tests with numeric strings and some other types (e.g. bool, float) to see what happens.